### PR TITLE
Longest_Common_Subsequence in Dart

### PR DESCRIPTION
**Pull request by JainStuti25**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Longest Common Subsequence
- 

#### Languages Used:
- Dart
- 

#### Files Added:
- Longest_Common_Subsequence.dart
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by JainStuti25** [comment]
@jainaman224 Please review my PR

**Comment by asha15** [comment]
please mention the issue ID in PR  template

File: Spiral_Array/Spiral_Array.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+def spiralPrint(m, n, a) :
```
Comment (asha15) [inline Spiral_Array/Spiral_Array.py:1]: remove this file

Comment (JainStuti25) [reply Spiral_Array/Spiral_Array.py]: Okay, I have removed this file.

File: Longest_Common_Subsequence/Longest_Common_Subsequence.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/*
+Problem Statement - Given two sequences, find the length of longest subsequence present in 
+both of them.A subsequence is a sequence that can be derived from another sequence by 
+deleting some or no elements without changing the order of the remaining elements.
+*/
+import 'dart:math';
+
+// I/P and O/P Examples
+void main() {
+    var inputSet1 = [10, 15, 20, 25, 30, 35, 40];
```
Comment (singh-shreya6) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.dart:10]: Please take dynamic inputs.

File: Longest_Common_Subsequence/Longest_Common_Subsequence.dart [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    int m = b.length;
+
+    var memo = new List<List<int>>.generate(n + 1,(_) => new List.filled(m+1, 0), growable: false);
+
+    for (int i = 0; i < a.length; i++) {
+        for (int j = 0; j < b.length; j++) {
+            // when the last character of both match, increase length of lcs by 1
+            if (a[i] == b[j]) {
+                memo[i+1][j+1] = memo[i][j] + 1;
+      }
```
Comment (singh-shreya6) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.dart:28]: Please indent the closing brace properly.

File: Longest_Common_Subsequence/Longest_Common_Subsequence.dart [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            if (a[i] == b[j]) {
+                memo[i+1][j+1] = memo[i][j] + 1;
+      } 
+            /* 
+            when the last character is not same, take maximum obtained by adding 
+            one character to one of the subsequences.
+            */
+            else {
+                memo[i+1][j+1] = max(memo[i+1][j], memo[i][j+1]);
+      }
```
Comment (singh-shreya6) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.dart:35]: Indentation incorrect for closing braces.

**Comment by singh-shreya6** [comment]
Are you working on it?

**Comment by singh-shreya6** [comment]
No updates on this one, so I am closing it. Please make changes and raise a new PR, if you want to work on this.

**Pull request closed without merging**